development/zh-cn/architecture-design.md

```diff
@@ -131,12 +131,12 @@
 DolphinScheduler使用ZooKeeper分布式锁来实现同一时刻只有一台Master执行Scheduler，或者只有一台Worker执行任务的提交。
 1. 获取分布式锁的核心流程算法如下
  <p align="center">
-   <img src="https://analysys.github.io/easyscheduler_docs_cn/images/distributed_lock.png" alt="获取分布式锁流程"  width="50%" />
+   <img src="/img/architecture-design/distributed_lock_zh_cn.png" alt="获取分布式锁流程"  width="55%" />
```

Comment:
Please rename to `distributed_lock_ch_zn.png`

Reply:
ok,i will modify it.

Reply:
@Tandoy Sorry, please rename the file name to zh_cn, it's my mistake 😭

Reply:
ok,i will fix them~